UPDATING.md

```diff
@@ -138,7 +138,7 @@ Type "help", "copyright", "credits" or "license" for more information.
 
 #### Template path of the file_task_handler
 
-The `file_task_handler` logger is more flexible. You can change the default format, `{dag_id}/{task_id}/{execution_date}/{try_number}.log` by supplying Jinja templating in the `FILENAME_TEMPLATE` configuration variable. See the `file_task_handler` for more information.
+The `file_task_handler` logger is more flexible. The default format can be changed, `{dag_id}/{task_id}/{execution_date}/{try_number}.log` by supplying Jinja templating in the `FILENAME_TEMPLATE` configuration variable. See the `file_task_handler` for more information.
```

Comment:
`The `file_task_handler` logger is more flexible` --> `The `file_task_handler` logger has been made more flexible`